### ISSUE TITLE
[BUG] no sandbox under Ubuntu 24.04

### ISSUE THREAD

**Issue by Tacitoku** [oxen-io/session-desktop#3244]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After updating the OS to Ubuntu 24.04 and executing the latest Session AppImage from the commandline, it terminates with this message: 
`[22482:1116/152451.248313:FATAL:zygote_host_impl_linux.cc(127)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/main/docs/linux/suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.`
on the [referenced site](https://chromium.googlesource.com/chromium/src/+/main/docs/linux/suid_sandbox_development.md) one reads:
`IMPORTANT NOTE: The Linux SUID sandbox is almost but not completely removed. See https://bugs.chromium.org/p/chromium/issues/detail?id=598454 This page is mostly out-of-date.`

Starting the AppImage with the option `--no-sandbox` works, but is not a secure method of use.
Moreover the external dependence is outdated. 

A convenient solution from the user's point of view would be an installation program that checks the dependencies on the target operating system and then installs everything necessary.

### Expected Behavior

The Session UI should start.

### Steps To Reproduce

1. install Ubuntu 24.04
2. download the Session AppImage for this OS
3. start the AppImage from the commandline
4. start the AppImage with the option `--no-sandbox`  from the commandline

### Desktop Version

session-desktop-linux-x86_64-1.14.2.AppImage

**Comment by Bilb** [comment]
Hey, could you try using the deb package instead and let me know if that works?

**Comment by Tacitoku** [comment]
Thanks for the hint, which solves the issue.
At first I searched on https://getsession.org/download for the instruction to install session-desktop using the dep package. 
Then I found the link to https://deb.oxen.io/ on https://github.com/oxen-io/session-desktop where I finally found the commands to install using the dep package:
```
    sudo curl -so /etc/apt/trusted.gpg.d/oxen.gpg https://deb.oxen.io/pub.gpg
    echo "deb https://deb.oxen.io $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/oxen.list
    sudo apt update
    sudo apt install session-desktop
```
And now session-desktop works great again on Ubuntu 24.04! 

Would it be good idea to inform the user who want to install session-desktop for Linux on https://getsession.org/download that there is an alternative way besides downloading the AppImage?

**Comment by KeeJef** [comment]
Yeah we should add a button to the website which links to instructions on how to install the .deb ( alongside the appimage button), tracking internally via WEB-350
